content/hi/docs/tutorials/configuration/configure-java-microservice/configure-java-microservice.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "माइक्रोप्रोफाइल, कॉन्फिगमैप्स और सीक्रेट्स का उपयोग करके कॉन्फिगरेशन को बाह्यीकृत करना"
+content_type: ट्यूटोरियल
+weight: 10
+---
+
+<!-- overview -->
+
+इस ट्यूटोरियल में आप सीखेंगे कि अपने माइक्रोसर्विस के कॉन्फ़िगरेशन को कैसे और क्यों लागू करना है। 
```

Comment:
लागू करना -> बाह्यीकृत करना 
@anushatomar13  please in entire page here Whether this is used everywhere or not.

Reply:
Done, it was only missing here and rest everywhere, I've replaced